packages/spectrum/test/util.tsx

```diff
@@ -34,20 +34,39 @@ import {
 } from '@jsonforms/core';
 import { JsonForms } from '@jsonforms/react';
 import { spectrumRenderers } from '../src/index';
+import {
+  defaultTheme,
+  Provider as SpectrumThemeProvider,
+} from '@adobe/react-spectrum';
+import { act } from 'react-dom/test-utils';
 
-export const mountForm = (
+type OnChangeType<T> = (change: { errors: any; data: T }) => void;
+
+// tslint:disable-next-line:only-arrow-functions
+export function mountForm<T extends object>(
   uischema: UISchemaElement,
   schema: JsonSchema = {},
-  data: any = {}
-): ReactWrapper => {
+  data?: T,
+  cells: any = [],
+  onChange: OnChangeType<T> = () => undefined
+): ReactWrapper {
   return mount(
-    <JsonForms
-      schema={schema}
-      uischema={uischema}
-      data={data}
-      renderers={spectrumRenderers}
-    />
+    <SpectrumThemeProvider theme={defaultTheme} scale='medium'>
+      <JsonForms
+        schema={schema}
+        uischema={uischema}
+        data={data ?? {}}
+        renderers={spectrumRenderers}
+        cells={cells}
+        onChange={onChange}
+      />
+    </SpectrumThemeProvider>
   );
+}
+
+export const simulateClick = (button: ReactWrapper) => {
+  const prop = button.prop<Function>('onPress');
+  act(() => prop());
 };
```

Comment:
Brauchts diese Funktion noch?

Reply:
yes - in the table array control tests, e.g. https://github.com/headwirecom/jsonforms-react-spectrum-renderers/blob/feature/26-tables/packages/spectrum/test/renderers/SpectrumTableArrayControl.test.tsx#L273